### ISSUE TITLE
Visualise models as graphs

### ISSUE THREAD

**Issue by rhayes777** [rhayes777/PyAutoFit#754]
Visualise model and prior relationships using a graph. Also visualise factor and variable relationships using a graph.

**Comment by rhayes777** [comment]
Yeah I recall although wasn't it a bit fiddly to get a clear graph?

**Comment by matthewghgriffiths** [comment]
Yeah, perhaps something like pyvis could be nice - https://towardsdatascience.com/pyvis-visualize-interactive-network-graphs-in-python-77e059791f01

**Comment by rhayes777** [comment]
Yeah that looks nice

**Comment by Jammy2211** [comment]
The first steps towards this are here: https://github.com/rhayes777/PyAutoFit/pull/923

I have put up a number of issues which aim to make model text output more concnise:

https://github.com/rhayes777/PyAutoFit/issues/925
https://github.com/rhayes777/PyAutoFit/issues/926
https://github.com/rhayes777/PyAutoFit/issues/930
https://github.com/rhayes777/PyAutoFit/issues/931

It goes without saying that the code makes model text output more concise, and which offers the right constructs for a visual graph, are likely one and the same, and all issues should be considered together.
